ReactWindows/ReactNative/Modules/Internationalization/InternationalizationModule.cs

```diff
@@ -0,0 +1,65 @@
+﻿using ReactNative.Bridge;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace ReactNative.Modules.Internationalization
+{
+    /// <summary>
+    /// A module that allows JS to get/set right to left preferences.
+    /// </summary>
+    class InternationalizationModule : NativeModuleBase
```

Comment:
And In-ter-na-tion-al-i-za-tion has more syllables than I-eight-teen-N, so it's harder to read :-)

---
In reply to: [76422044](https://github.com/ReactWindows/react-native-windows/pull/640#discussion_r76422044) [](ancestors = 76422044)